### DISCUSSION TITLE
Broken link on Modal component docs page

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#911] [General] [answered]
There is a broken link in the [Modal page ](https://paste.twilio.design/components/modal/ ). 
<img width="416" alt="Screen Shot 2020-11-04 at 2 52 44 PM" src="https://user-images.githubusercontent.com/67388401/98176639-c6e4f000-1ead-11eb-9999-3e46e568a5ec.png">

**Comment by aayushpi** [comment, marked as answer]
Thank you @sang-yeo. That component is not available yet, and I've filed a PR to fix it! For now, if you need to interrupt an action, use a modal. I request you to ask your engineers to follow with an upgrade to the alert dialog, when it's available. Visually, they'll be almost identical.

**Reply by sang-yeo** [reply]
Thanks! will do.
